snse/src/main/java/com/blakebarrett/snse/SentimentListActivity.kt

```diff
@@ -148,10 +152,11 @@ class SentimentListActivity : AppCompatActivity() {
 
         override fun getItemCount() = values.size
 
+        // TODO: Wire these back up.
         inner class ViewHolder(view: View) : RecyclerView.ViewHolder(view) {
-            val idView: TextView = view.id_text
-            val contentView: TextView = view.content
-            val backgroundView: View = view.id_background
+            val idView: TextView = TextView(parentActivity.baseContext) //view.id_text
+            val contentView: TextView = TextView(parentActivity.baseContext) //view.content
+            val backgroundView: View = View(parentActivity.baseContext) //view.id_background
```

Comment:
**TODO**: 
 1. Wire these into the argument `View` used by the `ViewHolder`.  This was just a stop-gap measure to get it compiling.
 2. Figure out how to consume a generated `*Binding` class for a `RecyclerView.ViewHolder`.